packages/@ember/-internals/glimmer/tests/integration/application/debug-render-tree-test.ts

```diff
@@ -99,7 +99,7 @@ if (ENV._DEBUG_RENDER_TREE) {
           this.outlet({
             type: 'route-template',
             name: 'index',
-            args: { positional: [], named: { model: undefined } },
+            args: { positional: [], named: { model: undefined }, errors: {} },
```

Comment:
We should probably change how this works: https://github.com/glimmerjs/glimmer-vm/commit/626b0e7caec9e6bff60752be71996e783eaeb55b#r132314108

Reply:
as an update, we reverted the change in glimmer-vm to later be re-implemented post-error-recovery PR